utils/time.py

```diff
@@ -1,11 +1,16 @@
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
 
+time_format = "%m/%d/%Y %H:%M:%S"
 
 class Time:
     @staticmethod
     def format_date(date):
-        return date.strftime("%m/%d/%Y %H:%M:%S") if date else None
+        return date.strftime(time_format) if date else None
+    
+    @staticmethod
+    def serialized_date_format(date):
+        return date.strftime("%Y-%m-%dT%H:%M:%S.%f")
```

Comment:
Perhaps we can do this, and remove all of the conditional logic that is wrapped around calling this method?
```suggestion
        return date.strftime("%Y-%m-%dT%H:%M:%S.%f") if date else None
```

Reply:
Oh duh! Good idea